index.html

```diff
@@ -221,9 +225,85 @@ <h1>Skills</h1>
                 <li>gitlab</li>
                 <li>Terminal</li>
               </ul>
-            </div>
           </section>
-          </div>
-          </body>
-          </html>
-         
\ No newline at end of file
+          <section class="touch">
+            <h2>
+              I'm always interested in hearing about new projects, so if you'd like
+              to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+              <ul class="touchForm">
+                <li>
+                  <input
+                    type="text"
+                    name="name"
+                    class="input-Text"
+                    placeholder="Full name"
+                    required
+                    maxlength="30"
+                    type="text"
+                  />
+                </li>
+                <li>
+                  <input
+                    class="input-Text"
+                    type="email"
+                    name="email"
+                    placeholder="Email address"
+                    required
+                    type="email"
+                  />
+                </li>
+                <li>
+                  <textarea
+                    class="inputTextarea"
+                    name="opinion"
+                    id="opinion"
+                    cols="50"
+                    rows="10"
+                    placeholder="Enter text here"
+                    maxlength="500"
+                    required
+                    type="text"
+                  ></textarea>
+                </li>
+                <li>
+                  <button type="button" class="touch-button">Get In Touch</button>
+                </li>
+              </ul>
+            </form>
+          </section>
+          <section class="footer">
+            <footer>
+              <ul class="Social-Media">
+                <li>
+                  <a href="https://github.com/ibtisam34">
+                    <img src="images/github.png" alt="-GitHub" />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://www.linkedin.com/in/riyangwl2">
+                    <img src="images/linkedn.png" alt="Linkedin" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img src="images/angelist.png" alt="Angelist" />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://twitter.com/Queenjin12">
+                    <img src="images/twitter.png" alt="Twitter" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img src="images/medium.png" alt="Medium" />
+                  </a>
+                </li>
+              </ul>
+            </footer>
```

Comment:
- [ ] I kindly suggest that you add the horizontal line from the Figma design below:

| Your Design  | Figma Design |
| ------------- | ------------- |
|<img width="221" alt="image" src="https://user-images.githubusercontent.com/52934128/194043561-e7070967-2ad4-4a30-902e-6b07f8740209.png"> | Content Cell  |
| Content Cell  | ![image](https://user-images.githubusercontent.com/52934128/194043314-c6e5e37e-99bf-4976-840b-bc7de395f6c3.png)  |

Reply:
@Ibtisam34 , I have edited the table to make it clearer. Please kindly add the horizontal line.